suomen-vaylat-app/src/components/measurement-tools/DrawingTools.jsx

```diff
@@ -254,9 +254,9 @@ export const DrawingTools = ({isOpen}) => {
         // stop the drawing tool
         channel && channel.postRequest('DrawTools.StopDrawingRequest', [activeTool]);
         store.dispatch(setGeoJsonArray([]));
-        store.dispatch(removeFromDrawToolMarkers());
+        store.dispatch(removeFromDrawToolMarkers(false));
```

Comment:
Eikö markkereita haluta poistaa jos piirtotyökalu suljetaan? Jos on näin niin kaikki ok

Reply:
Ilmeisesti ajatuksissani muuttanut tuota. Muutettu nyt takaisin trueksi